### PR TITLE
Source NAP Peilmerken from Amsterdam Schema, add migration and migration utils

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

File: src/tests/test_alembic_utils.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class TestAlembicUtils(TestCase):
+    def test_get_query_split_json_column(self):
+        result = get_query_split_json_column('nap_peilmerken', 'status', {'code': 'status_code', 'omschrijving': 'status_omschrijving'}, {'code': 'int', 'omschrijving': 'varchar'})
+        expected = "UPDATE nap_peilmerken SET status_code=(status->>'code')::int, status_omschrijving=(status->>'omschrijving')::varchar"
+        self.assertEquals(expected, result)
+
+    def test_get_query_merge_columns_to_jsonb_column(self):
+        result = get_query_merge_columns_to_jsonb_column('nap_peilmerken', 'merk', {'code': 'merk_code', 'omschrijving': 'merk_omschrijving'})
+        expected = "UPDATE nap_peilmerken SET merk = jsonb_build_object('code', \"merk_code\", 'omschrijving', \"merk_omschrijving\")"
+        self.assertEquals(expected, result)
```
Comment (RoelWKramer) [inline src/tests/test_alembic_utils.py:15]: add trailing newline (linting should catch this?)

```suggestion
        self.assertEquals(expected, result)
        
```

Comment (JKoetsier) [reply src/tests/test_alembic_utils.py]: Accepted the change. But add it to the linter if you think that it's necessary to have the trailing newline ;)

File: src/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
 alembic==1.7.5
--e git+https://github.com/Amsterdam/GOB-Core.git@v0.17.20#egg=gobcore
```
Comment (RoelWKramer) [inline src/requirements.txt:2]: v1.0.1 not required?

Comment (JKoetsier) [reply src/requirements.txt:2]: Nope

**Review by RoelWKramer — APPROVED** [review]

**Pull request merged**